map.py

```diff
@@ -100,11 +95,14 @@ def generate_mapping_information(num_clients: int) -> tuple[nx.MultiDiGraph, int
     return map_network, depot_id, client_subset, map_bounds
 
 
-def _get_nodes(G: nx.Graph, force_id: int) -> tuple[folium.CustomIcon, list[int]]:
+def _get_node_info(
+    G: nx.Graph, node_id: int, icon_name: str
+) -> tuple[folium.CustomIcon, list[int]]:
     """Get node demand values and icons for each client location."""
-    location_icon = folium.CustomIcon(str(force_icon_path), icon_size=(32, 37))
-    sources = ("demand_water", "demand_food", "demand_other")
-    return location_icon, [G.nodes[force_id][s] * 100 for s in sources]
+    icon_path = Path(__file__).parent / f"assets/location_icons/{icon_name}.png"
+    location_icon = folium.CustomIcon(str(icon_path), icon_size=(30, 48))
+    sources = (f"resource_{i}" for i in range(len(RESOURCES)))
+    return location_icon, [G.nodes[node_id][s] * 100 for s in sources]
```

Comment:
Not that it matters much, but I feel like the `... * 100` would make more sense when adding the resources above rather than here.

Reply:
Yeah I thought so too but I assumed there was a reason it was done this way. I'll try moving it and see if anything breaks.

Reply:
updated: https://github.com/dwave-examples/mvrp/pull/11/commits/b87cd34c7b40ca88d095e4bf9107cb5c7e87060d